### DISCUSSION TITLE
What should I consider when cleaning/deleting duplicate records?

### DISCUSSION THREAD

**Discussion by magpiedin** [fieldmuseum/EMu-Documentation#939] [FAQs]
### When cleaning duplicates: 
If you're not sure which record to keep, consider the following:

1. **Which records are published online, if any?**
    - On the record's Security tab -- does 'Publish on Internet" = "Yes"?
    - If "Yes" that record might be publicly referenced by other websites or researchers, so you might want to keep it.
    - If multiple records are published, consider **# 2** below:

2. **Which records are attached to Catalog or other records?**
    - In a duplicate record, go to View -> Attachments -> Options -> Show Tabs
    - Go to the 'Objects' tab -- Are any Catalog records listed there?
    - If multiple records are public (or otherwise similar), consider keeping the record with the most attachments.
      - Note: If a record is attached to Catalog or other records, it needs to be detached first. If it is a duplicate that also needs to be replaced by a consolidated record, use the ['Merge' tool](https://github.com/fieldmuseum/EMu-Documentation/discussions/389) to properly detach & replace its attachments.

3. **If all else is equal** (attachments, publish settings, and other values in the records), consider keeping the oldest record -- with the lowest IRN.

To delete EMu records, select them in a list view by clicking on their numbers (so the rows turn blue), and go to File -> Delete -> Selected Records 

#### Multimedia-specific recommendations:
- https://github.com/fieldmuseum/EMu-Documentation/discussions/938
- https://github.com/fieldmuseum/EMu-Documentation/discussions/384

**Comment by rondlg** [comment]
EMu is very good at creating duplicates but terrible at giving us ways to disambiguate them. Here are all the duplicate(s) questions so far!

### erights
- https://github.com/fieldmuseum/EMu-Documentation/discussions/1081


### ecollectionevents
- https://github.com/fieldmuseum/EMu-Documentation/discussions/900

### emultimedia
- https://github.com/fieldmuseum/EMu-Documentation/discussions/384
- https://github.com/fieldmuseum/EMu-Documentation/discussions/771
-
